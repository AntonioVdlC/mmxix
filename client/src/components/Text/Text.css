@import "../../styles/vars.css";

.text {
  display: block;
  width: 100%;
  margin: 42px;
  text-align: center;

  font-size: var(--font-size-xxxl);
  color: var(--color-complementary);
  font-family: var(--font-family-secondary);
}