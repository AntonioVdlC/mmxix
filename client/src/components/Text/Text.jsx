import React from "react";
import PropTypes from "prop-types";

import styles from "./Text.css";

const Text = ({ text }) => <span className={styles.text}>{text}</span>;

Text.propTypes = {
  text: PropTypes.string.isRequired
};

export default Text;
