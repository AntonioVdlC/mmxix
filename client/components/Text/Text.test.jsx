import React from "react";
import renderer from "react-test-renderer";

import Text from "./Text";

describe("Text", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<Text text="Hello, world!" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
