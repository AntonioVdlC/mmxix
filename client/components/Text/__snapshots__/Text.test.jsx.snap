// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text should render correctly 1`] = `
<span
  className="text"
>
  Hello, world!
</span>
`;
